mlir/utils/jenkins/Jenkinsfile

```diff
@@ -341,10 +341,12 @@ boolean shouldRunFromChip(String chip) {
             return shouldRunFromCodepath("vanilla")
         case "gfx90a":
             // Special case because all our "vanilla" hosts are gfx90a.
-            return shouldRunFromCodepath("mfma") || shouldRunFromCodepath("vanilla")
+            return params.disable90a == false &&
```

Comment:
Added disable switches for the other architectures, if we want to focus on a single one for debugging.